src/main/resources/application.properties

```diff
@@ -2,6 +2,8 @@ spring.datasource.url=jdbc:postgresql://localhost:5432/Laborete
 spring.datasource.username=postgres
 spring.datasource.password=0625
 spring.jpa.hibernate.ddl-auto=create-drop
+spring.jpa.show-sql=true
```

Comment:
Check application.properties carefully before including them to a merge request.
Sometimes, you can commit local configuration.
Personally, I am not sure if we need added two properties.
@Tony001-HUB Could you please clarify?